src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_StrandBiasTestUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator.allelespecific;
+
+import htsjdk.variant.variantcontext.*;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class AS_StrandBiasTestUnitTest extends GATKBaseTest {
+
+    private static final Allele REF = Allele.create("T", true);
+    private static final Allele ALT = Allele.create("A", false);
+
+    /**
+     * Test for issue #6766
+     */
+    @Test
+    public void testCombineRawData() {
```

Comment:
Give the test a more specific name: `testCombineRawDataNullPointerExceptionIssue6766()`